superset-frontend/src/common/components/Tabs.tsx

```diff
@@ -52,4 +52,10 @@ const Tabs = styled(BaseTabs)`
   }
 `;
 
+const StyledTabPane = styled(AntdTabs.TabPane)``;
```

Comment:
Is this styled component intended to be empty?

Reply:
Yeah... just a placeholder/scaffolding, just to set-up the pattern. I expect I'll be adding some styles there in a "horizontal" pass soon.

Reply:
Some of the styles on the parent `Tabs` element can probably be moved into `TabPane` styling